charts/operator/templates/role.yaml

```diff
@@ -88,6 +88,11 @@ rules:
   apiGroups: ["apps"]
   resourceNames: ["rule-evaluator"]
   verbs: ["get", "patch", "update"]
+- resources:
+  - deployments/scale
```

Comment:
I'm confused...why add these permissions now, which are a subset of the permissions that already exist? Why not just scale back the existing permissions when the rest of the necessary changes are ready?

Reply:
I tried running this change without this permission and it failed. You would think they are a subset but that's not how it's implemented in Kubernetes...